frontend/src/components/Pagination/Pagination.jsx

```diff
@@ -49,14 +49,17 @@ function getPager(totalItems, currentPage) {
 function Pagination({ currentPage, totalItems, onChangePage }) {
   const [pager, setPager] = useState(getPager(totalItems, INITIAL_PAGE));
 
-  const changePage = newPage => {
-    if (newPage < 1 || newPage > pager.totalPages) {
-      return;
-    }
-    const newPager = getPager(totalItems, newPage);
-    setPager(newPager);
-    onChangePage(newPage);
-  };
+  const changePage = useCallback(
+    newPage => {
+      if (newPage < 1 || newPage > pager.totalPages) {
```

Comment:
I think we can have the opposite conditions and avoid the unnecessary return:
```suggestion
      if (newPage > 0 || newPage <= pager.totalPages) {
      const newPager = getPager(totalItems, newPage);
      setPager(newPager);
      onChangePage(newPage);
      }
```

Reply:
Totally right!